torch/__init__.py

```diff
@@ -342,3 +342,11 @@ def compiled_with_cxx11_abi():
 
 # Import the quasi random sampler
 import torch.quasirandom
+
+# Register at fork handler to initialize OpenMP in child processes (see gh-28389)
+def _torch_at_fork(self):
+    torch.get_num_threads()
+
+import multiprocessing as _mp
+_mp.util.register_after_fork(_torch_at_fork, _torch_at_fork)
```

Comment:
mp.util is not automatically imported. so you should do something like `from multiprocessing.util import register_after_fork`.

Reply:
CI is still failing after making that change. Looks like it's actually a python 2 issue.

I suppose this will need to be done using `pthread_atfork` in c++ unless there's a way to do it in python 2.

Reply:
The handler definitely is a thing in python 2, see https://github.com/pytorch/pytorch/blob/master/torch/multiprocessing/reductions.py#L8. I think it is the python 2 import system. Likely a `from __future__ import absolute_import` will fix.

Reply:
Okay, we can see if that fixes it.